Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -0,0 +1,57 @@
+import KsApi
+import Library
+import UIKit
+
+internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControllerDataSource {
+  private let viewControllers: [UIViewController]
+
+  internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
```

Comment:
could we use `BackerDashboardProjectsViewControllerDelegate` as the type of `delegate` instead of `UIViewController`?